bin/automatic_review.py

```diff
@@ -0,0 +1,250 @@
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from analysis_driver import rest_communication
+from analysis_driver.constants import ELEMENT_REVIEW_COMMENTS
+
+
+
+class RunReview():
+    def __init__(self, run_name):
+        self.run_name = run_name
+        self.run = self.read_run()
+        self.lane = self.read_lanes()
+
+    def read_run(self):
+        return rest_communication.get_documents(
+            'aggregate/all_runs',
+            match={"run_id":self.run_name}
+        )
+
+    def read_lanes(self):
+        return rest_communication.get_documents(
+            'aggregate/run_elements',
+            match={"run_id":self.run_name}
+        )
+
+    def run_metrics(self):
+        for r in self.run:
+            proc_status = str(r.get("proc_status"))
+            return proc_status
+
+    def lanes_metrics(self):
+        lane_metrics = {}
+        for r in self.lane:
+            metrics_per_lane = {}
+            lane = str(r.get('lane'))
+            metrics_per_lane['yield_in_gb'] = str(r.get('yield_in_gb'))
+            metrics_per_lane['pc_q30'] = str(r.get('pc_q30'))
+            metrics_per_lane['pc_pass_filter'] = str(r.get('pc_pass_filter'))
+            lane_metrics[lane] = metrics_per_lane
+        return lane_metrics
+
+    def run_review(self):
+        if self.run and self.lane:
+            proc_status = self.run_metrics()
+            lanes_metrics = self.lanes_metrics()
+            run_review = {}
+            for lane in lanes_metrics:
+                lane_review = {}
+                pc_pass_filter = (lanes_metrics[lane]).get('pc_pass_filter')
+                yield_in_gb = (lanes_metrics[lane]).get('yield_in_gb')
+                pc_q30 = (lanes_metrics[lane]).get('pc_q30')
+
+                if proc_status == 'finished':
+                    lane_review['proc_status'] = 'pass'
+                else:
+                    lane_review['proc_status'] = 'fail'
+
+                if float(yield_in_gb) > 960:
+                    lane_review['yield_in_gb'] = 'pass'
+                else:
```

Comment:
Look at column_mappings.yaml in reporting app - organise parameters and thresholds similarly